jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/jobs/HostAssetDataImporter.java

```diff
@@ -123,15 +123,15 @@ public Map<String, Object> execute() {
             errorMap.put(ERROR_TYPE, FATAL);
             errorMap.put(EXCEPTION, e.getMessage());
             errorList.add(errorMap);
-            List<PermissionVH> permissionVHList=new ArrayList<>();
-            PermissionVH permissionVH=new PermissionVH();
-            ErrorVH errorVH=new ErrorVH();
-            errorVH.setException(e.getMessage());
-            errorVH.setType("ec2");
-            permissionVH.setAccountNumber("Qualys-connector");
-            permissionVH.setErrorVH(errorVH);
-            permissionVHList.add(permissionVH);
-            NotificationPermissionUtils.triggerNotificationsForPermissionDenied(permissionVHList,"Qualys");
+//            List<PermissionVH> permissionVHList=new ArrayList<>();
+//            PermissionVH permissionVH=new PermissionVH();
+//            ErrorVH errorVH=new ErrorVH();
+//            errorVH.setException(e.getMessage());
+//            errorVH.setType("ec2");
+//            permissionVH.setAccountNumber("Qualys-connector");
+//            permissionVH.setErrorVH(errorVH);
+//            permissionVHList.add(permissionVH);
+//            NotificationPermissionUtils.triggerNotificationsForPermissionDenied(permissionVHList,"Qualys");
```

Comment:
Please always remove commented code when the final PR is to master. Also, when working in a file, please spot check and remove commented code as part of general code cleanup routine.

Commented code leaves clutter and can always look into git history for previous implementation.

Reply:
Addressed in commit